deephyper/skopt/moo/_multiobjective.py

```diff
@@ -0,0 +1,145 @@
+import abc
+from unicodedata import is_normalized
+import numpy as np
+from deephyper.skopt.utils import is_listlike
```

Comment:
organise imports properly. On VScode you can do `cmd+shift+P` then select `organise imports`

Reply:
Very useful. I was not aware of this..!